tensorflow/python/kernel_tests/data_structures/tensor_array_ops_test.py

```diff
@@ -1784,6 +1786,16 @@ def testSkipEagerTensorArrayInt64GPU(self):
       v0, v1 = sess.run([r0, r1], feed_dict={value: [-3, 100]})
       self.assertAllEqual(v0, -3)
       self.assertAllEqual(v1, 100)
+  
+  @test_util.deprecated_graph_mode_only
+  def testTensorArrayScatterBfloat61GPU(self):
+    if not test.is_gpu_available():
+      return
+    with self.session(force_gpu=True) as sess:
+      ta = tensor_array_ops.TensorArray(
+          dtype=dtypes.float32, tensor_array_name="foo", size=5)
```

Comment:
Use bfloat16 here

Reply:
Thanks @reedwm for the review. Fixed this.